### PR TITLE
bluetooth-audio: initial add-on

### PR DESCRIPTION
**Pull request by chewitt**
This service add-on provides the ability to auto-switch the Kodi audio output device to pulseaudio when a BT audio device (speaker) is connected. Picked from @lrusak repo with permission.

### CODE REVIEW

File: packages/addons/service/bluetooth-audio/source/resources/language/English/strings.po [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 
+msgid ""
+msgstr ""
+
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
```
Comment (vpeter4) [inline packages/addons/service/bluetooth-audio/source/resources/language/English/strings.po]: This is some bad copy/paste?

**Comment by chewitt** [comment]
@vpeter4 strings.po is cleaned up

**Comment by mglae** [comment]
Kodi refuses loading strings.po files without header.

**Comment by chewitt** [comment]
@mglae please figure out what's missing and submit the PR along with a version bump :)

**Pull request merged**